packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -271,7 +271,7 @@ class FormAds extends Component {
             advertiser_notes: [
                 v => !!v,
                 v => v.length < 400,
-                v => /^[\p{L}\p{Nd}\s'.,:;()@#/-]{1,500}$/u.exec(v) !== null,
+                v => /^[\p{L}\p{Nd}\s'.,:;()@#+/-]{1,300}$/u.exec(v) !== null,
```

Comment:
if im not mistaken there is a hint on description that shows which symbols are allowed, can you change that too 🙏

Reply:
right, thanks :)